mail/mail-server/Portfile

```diff
@@ -0,0 +1,877 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               active_variants 1.1
+
+name                    mail-server
+version                 1.0
+revision                0
+categories              mail net
+platforms               darwin
+supported_archs         noarch
+maintainers             {ieee.org:s.t.smith @essandess} openmaintainer
+license                 GPL-3
+distfiles
+
+description             Mail server configuration
+long_description ${description} \
+    Mail server working configuration that provides a basic, working, \
+    easily modifiable mail server. The configuration is built using \
+    postfix for the MTA, dovecot for the MDA, solr for fast search, \
+    rspamd for a milter, and clamav for email virus scans. The \
+    configuration includes a surrogate TLS certificate and DKIM.
+
+homepage                https://www.postfix.org/ \
+                        https://www.dovecot.org/ \
+                        https://rspamd.com/
+
+set postfix_required_variants \
+                        [lsort {dovecot_sasl pcre smtputf8 tls}]
+set dovecot2_required_variants \
+                        [lsort {gssapi solr}]
+
+depends_lib-append      port:postfix \
+                        port:dovecot2 \
+                        port:dovecot2-sieve \
+                        port:rspamd \
+                        port:dcc \
+                        port:openssl \
+                        port:sf-pwgen
+
+# these dependencies are from variants that may not be active
+depends_lib-append      port:pcre \
+                        port:apache-solr8 \
+                        port:clucene \
+                        port:curl \
+                        port:expat
+
+depends_run-append      port:clamav-server \
+                        port:dns-server \
+                        port:logrotate
```

Comment:
Please sort the `depends_lib` and `depends_run` dependencies too.

Reply:
Also done. Thanks.